spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -207,6 +206,52 @@ def test_go_to_cursor_position(create_outlineexplorer, qtbot):
     assert outlineexplorer.treewidget.currentItem().text(0) == 'method1'
 
 
+def test_follow_cursor(create_outlineexplorer, qtbot):
+    """
+    Test that the cursor is followed
+    """
```

Comment:
This is not really a fix needed, but just a reminder that docstrings should be handle as complete sentences, hence we should try to add a period at the end ;-)

Reply:
Correct, per PEP 257.